src/codec_aom.c

```diff
@@ -748,7 +748,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
             cfg->g_lag_in_frames = 0;
         }
         if (encoder->maxThreads > 1) {
-            cfg->g_threads = encoder->maxThreads;
+            // libaom fails if cfg->g_threads is greater than 64 threads. See MAX_NUM_THREADS in
+            // aom/aom_util/aom_thread.h.
```

Comment:
This header is public right(encoder/encoder.h references it)? So maybe it should be included and MAX_NUM_THREADS be used (in case it changes one day).